web/src/components/TickerTable/index.tsx

```diff
@@ -31,7 +35,7 @@ export const TickerTable: React.FC<Props> = ({
                     </td>
                     <td>
                         <span>
-                            <Decimal fixed={market.amount_precision} thousSep=",">
+                            <Decimal fixed={!isFixedPrecision ? market.amount_precision : market.price_precision} thousSep=",">
```

Comment:
Market.last is market last price. For displaying price we use market.price_precision.